### PR TITLE
Syto 22

### PR DESCRIPTION
**Pull request by xiwcx**
https://jira.nationalgeographic.com/browse/SYTO-22

### CODE REVIEW

**Comment by jleja** [comment]
You can probably generalize the examples so they aren't straight out of Your Shot.  Also, consider expanding the callout triangle positions so they are all present and accounted for.  YS omitted a few because they weren't needed, but Mortar serves a bit of a different purpose.

File: app/src/styles/elements/_tooltip.scss [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+*/
+
+/*
+Tooltips:
+
+CSS-only implementation of tooltips.  Installed as part of the main
+Desktop CSS file.  To activate:
+
+- Place the class "tooltip-wrapper" on the element that you want serve as
```
Comment (chuckcarpenter) [inline app/src/styles/elements/_tooltip.scss]: This is name spaced, don't you mean 'mt_tooltip-wrapper'?

File: app/src/styles/elements/_tooltip.scss [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  the trigger.  This element should serve as the parent.
+
+- Place your contents somewhere inside the tooltip-wrapper element with the
+  class "tooltip <<direction>>" where direction = where the callout triangle
```
Comment (chuckcarpenter) [inline app/src/styles/elements/_tooltip.scss]: Again, not `mt_`?

File: app/src/styles/elements/_tooltip.scss [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+<div class="icons">
+    <ul>
+        <li class="ico-open mt_tooltip-wrapper">
+            <div class="ico"></div>
+            <p>Top</p>
+            <div class="mt_tooltip-tip mt_tooltip-tip--bottom">Everyone’s welcome to contribute their best shot to open assignments.  <a href='/about/#section2'>Learn more</a>.</div>
+        </li>
+        <li class="ico-closed mt_tooltip-wrapper">
+            <div class="ico"></div>
```
Comment (chuckcarpenter) [inline app/src/styles/elements/_tooltip.scss]: @davidocoulter noted to me, what's this for @xiwcx ? Don't see a style for it added and my initial thought was the arrow, but now I'm not sure.

Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: That is code that @xiwcx pulled from Your Shot when he asked for live examples of the tooltips.  Totally not necessary for tooltips.  And, really, the examples can be made much simpler and less YS-specific.

File: app/src/styles/elements/_tooltip.scss (hunk 1 above)
Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: That was lifted from where I implemented the tooltips in Your Shot.  Should be updated to be mortar-specific.

File: app/src/styles/elements/_tooltip.scss (hunk 2 above)
Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: See above.

File: app/src/styles/elements/_tooltip.scss [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        bottom: -21px;
+        left: 45%;
+    }
+
+    &:after {
+        border-top: 10px solid $mt_color-basic;
+        bottom: -20px;
+        left: 45%;
+    }
+}
```
Comment (davidocoulter) [inline app/src/styles/elements/_tooltip.scss]: Why do we need both :before and :after here? Wouldn't one or the other suffice?

File: app/src/styles/elements/_tooltip.scss [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+            <div class="ico"></div>
+            <p>Top</p>
+            <div class="mt_tooltip-tip mt_tooltip-tip--bottom">Everyone’s welcome to contribute their best shot to open assignments.  <a href='/about/#section2'>Learn more</a>.</div>
+        </li>
+        <li class="ico-closed mt_tooltip-wrapper">
+            <div class="ico"></div>
+            <p>Top left</p>
+            <div class="mt_tooltip-tip mt_tooltip-tip--bottom mt_tooltip-arrow--left">Completed assignments—with our favorite photos included—will be published online. <a href='/about/#section1'>Learn more</a>.</div>
+        </li>
+        <li class="ico-published mt_tooltip-wrapper">
```
Comment (davidocoulter) [inline app/src/styles/elements/_tooltip.scss]: What does "ico-published" do?

File: app/src/styles/elements/_tooltip.scss [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+            <div class="ico"></div>
+            <p>Top Right</p>
+            <div class="mt_tooltip-tip  mt_tooltip-tip--bottom mt_tooltip-arrow--right">Once the submission period is over, we'll review all contributions and select our favorite images to be included in the story. <a href='/about/#section2'>Learn more</a>.</div>
+        </li>
+        <li class="ico-open mt_tooltip-wrapper">
```
Comment (davidocoulter) [inline app/src/styles/elements/_tooltip.scss]: What does "ico-open" do?

File: app/src/styles/elements/_tooltip.scss [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+            <div class="ico"></div>
+            <p>Bottom</p>
+            <div class="mt_tooltip-tip mt_tooltip-tip--top">Everyone’s welcome to contribute their best shot to open assignments.  <a href='/about/#section2'>Learn more</a>.</div>
+        </li>
+        <li class="ico-closed mt_tooltip-wrapper">
```
Comment (davidocoulter) [inline app/src/styles/elements/_tooltip.scss]: What does "ico-closed" do?

File: app/src/styles/elements/_tooltip.scss (hunk 4 above)
Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: You're basically overlaying 2 triangles over each other.  One for the "border" and one to sit inside it but be slightly smaller to give the border effect.

File: app/src/styles/elements/_tooltip.scss (hunk 6 above)
Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: In Your Shot, it displays an open icon that applies to Assignments.  In Mortar, it does nothing.

File: app/src/styles/elements/_tooltip.scss (hunk 5 above)
Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: In Your Shot, it displays a published icon that applies to Assignments.  In Mortar, it does nothing.

File: app/src/styles/elements/_tooltip.scss (hunk 7 above)
Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: Ico-closed some something very special.  But only on the 3rd Sunday after the Summer Solstice and only for IE8 browsers.  All others will just show a closed icon in Your Shot that applies to Assignments.

Comment (davidocoulter) [inline app/src/styles/elements/_tooltip.scss]: I'm guessing that here, it does nothing.

Comment (jleja) [inline app/src/styles/elements/_tooltip.scss]: That is correct, sir.

**Pull request merged**